packages/elements-core/src/components/Docs/HttpOperation/HttpOperation.tsx

```diff
@@ -126,20 +125,14 @@ function MethodPath({ method, path }: MethodPathProps) {
 function MethodPathInner({ method, path, chosenServerUrl }: MethodPathProps & { chosenServerUrl: string }) {
   const isDark = useThemeIsDark();
   const fullUrl = `${chosenServerUrl}${path}`;
-  const { leading, trailing } = useChosenServerUrl(chosenServerUrl);
 
   const pathElem = (
     <Flex overflowX="hidden" fontSize="lg" userSelect="all">
       <Box dir="rtl" color="muted" textOverflow="truncate" overflowX="hidden">
-        {leading}
-
-        {trailing !== null && (
-          <Box as="span" dir="ltr" style={{ unicodeBidi: 'bidi-override' }}>
-            {trailing}
-          </Box>
-        )}
+        <Box as="span" dir="ltr" style={{ unicodeBidi: 'bidi-override' }}>
+          {chosenServerUrl}
+        </Box>
       </Box>
```

Comment:
@P0lip I'm not quite sure we strictly speaking needed to do the potential split / re-stich here, if I'm understanding the initial issue.  I do see that if you don't pull out the trailing non-alpha stuff it gets reshuffled to the front if you use rtl ordering.

But I *think* just wrapping the entire url inside a ltr inside the outer rtl Box accomplishes what you were initially fixing here:  https://github.com/stoplightio/elements/pull/2217 - ie, I still get the desired behavior of ellipsis at the front of the url, not the back - and it still maintains urls like 'https:/something.com--' - and also is still selectable.

Anyhow, take a look, let me know what I'm missing and broke 🤣